src/main/java/frc/robot/CommandLogger.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
We can replace this and all references to it with the new command logger taken from main